docs/Dialects/FIRRTL/FIRRTLAnnotations.md

```diff
@@ -299,6 +299,31 @@ Example:
 }
 ```
 
+### Convention
+
+| Property   | Type   | Description                 |
+| ---------- | ------ | --------------------------- |
+| class      | string | `firrtl.module.Convention`  |
+| convention | string | `scalarize` or `internal`   |
+| target     | string | Reference target            |
```

Comment:
It seems like one should have target strings for modules, but the Chisel way is to have this information on instances.  @seldridge should comment on implementability on the chisel side.